specs/deneb/light-client/fork.md

```diff
@@ -43,6 +43,7 @@ def upgrade_lc_header_to_deneb(pre: capella.LightClientHeader) -> LightClientHea
             withdrawals_root=pre.execution.withdrawals_root,
             data_gas_used=uint64(0),  # [New in Deneb:EIP4844]
             excess_data_gas=uint64(0),  # [New in Deneb:EIP4844]
+            parent_beacon_block_root=Root(),  # [New in Deneb:EIP4788]
```

Comment:
wasn't entirely sure how to handle this, as we could also grab the parent block root from `pre`

@etan-status perhaps you know the best route here?